.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+    name: cargo check/clippy/fmt/test
+    runs-on: ubuntu-latest
+
+    env:
+      CARGO_TERM_COLOR: always
+      RUST_BACKTRACE: 1
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: rustup toolchain install stable --profile minimal
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          # bump this to invalidate build/dependency cache
+          prefix-key: "v0-rust"
```

Comment:
Manual bumping is prone to errors. Can the key be based on the Cargo.lock hash?

Reply:
Yes, it is keyed by the `rustc` version, `Cargo.lock`/`Cargo.toml` hashes, and some other things as [explained here](https://github.com/Swatinem/rust-cache#cache-details).
The `prefix-key` still allows us to manually invalidate the cache when there is something wrong with the action.
I'll update the comment to make that clear.